client/dashboard/sites/overview/site-card.tsx

```diff
@@ -70,10 +74,14 @@ export default function SiteCard( {
 						<Field title={ __( 'Status' ) }>{ getSiteStatusLabel( site ) }</Field>
 					</HStack>
 					<HStack justify="space-between">
-						{ options?.software_version && (
-							<Field title={ __( 'WordPress' ) }>{ options.software_version }</Field>
+						<Field title={ __( 'WordPress' ) }>
+							{ options?.software_version ?? __( 'Unknown' ) }
+						</Field>
+						{ is_wpcom_atomic && (
```

Comment:
Why are we checking `is_wpcom_atomic` and not something like "feature php version available". (I think we should move away from checking "type of sites" in the different dashboard components as much as we can.

Reply:
I'm not sure I understand 🤔

Reply:
Ok, I guess this will have to be added to the API

Reply:
Yeah, just to be clear (discussed this in Slack with other folks). This is more like a "vision" at this point and not a blocking comment.